doc/man7/provider.pod

```diff
@@ -147,6 +147,30 @@ The number for this operation is B<OSSL_OP_KEYEXCH>.
 The functions the provider can offer are described in
 L<provider-keyexch(7)>
 
+=item Asymmetric Ciphers
+
+In the OpenSSL libraries, the corresponding method object is
+B<EVP_ASYM_CIPHER>.
+The number for this operation is B<OSSL_OP_ASYM_CIPHER>.
+The functions the provider can offer are described in
+L<provider-asym_cipher(7)>
+
+=item Signatures
+
+In the OpenSSL libraries, the corresponding method object is
+B<EVP_SIGNATURE>.
+The number for this operation is B<OSSL_OP_SIGNATURE>.
+The functions the provider can offer are described in
+L<provider-signature(7)>
+
+=item Key Management
+
+In the OpenSSL libraries, the corresponding method object is
+B<EVP_KEYMGMT>.
+The number for this operation is B<OSSL_OP_KEYMGMT>.
+The functions the provider can offer are described in
+L<provider-keymgmt(7)>
```

Comment:
It might be worth saying already here that this type is seldom used alone, that it's normally paired with methods that use asymmetric keys, i.e. EVP_KEYEXCH, EVP_ASYM_CIPHER and EVP_SIGNATURE

Reply:
Well, it was actually writing this section that inspired we to raise the question in #11350. Because if we decide that they shouldn't be public at all then we'd need to tweak the wording again. However for the purposes of this PR I thought it should still be mentioned. I can add some words along the lines you suggest.

Reply:
If those types are made non-public, *they shouldn't be mentioned at all in public documentation*

Reply:
> If those types are made non-public, they shouldn't be mentioned at all in public documentation

Oh, yes I agree. But as right now they are public, they should be mentioned. If at some later point in time they are no longer public then I expect a later PR to come and change this wording again.

Reply:
Deal!